Makefile

```
@@ -24,21 +25,18 @@ check:
 	find . -name "*.pyc" -exec rm {} \;
 	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --verbose --showlocals $(TESTS_TARGET)
 
+requre_data_cleanup:
+	requre-patch purge --replaces "copr.v3.helpers:login:str:somelogin" --replaces "copr.v3.helpers:token:str:sometoken" $(TESTS_RECORDING_PATH)/test_data/*/*yaml
+	requre-patch create-symlinks $(CURDIR)/$(TESTS_RECORDING_PATH)/test_data/
+
 # example: TESTS_TARGET=tests/unit/test_api.py make check_in_container
 check_in_container: tests_image
-	$(CONTAINER_ENGINE) run --rm -ti \
-		--env TESTS_TARGET \
-		-v $(CURDIR):/src \
-		--security-opt label=disable \
-		$(TESTS_IMAGE) \
-		bash -c "pip3 install .; make check"
+	$(CONTAINER_COMMON) $(CONTAINER_WHAT)
 
 # Mounts your ~/.config/ where .packit.yaml with your github/gitlab tokens is expected
+# Mounts ssh connfig dir, to have ssh keys for fedpkg cloning
+# create random tmpdir and mount into /tmp to avoid issues with creating temporary dirs via python
 check_in_container_regenerate_data: tests_image
-	$(CONTAINER_ENGINE) run --rm -ti \
-		--env TESTS_TARGET=$(TESTS_RECORDING_PATH) \
-		-v $(CURDIR):/src \
-		-v $(HOME)/.config:/root/.config \
-		--security-opt label=disable \
-		$(TESTS_IMAGE) \
-		bash -c "pip3 install .; make check"
+	$(eval RANDOM_TMP_DIR = $(shell mktemp -d))
+	$(CONTAINER_COMMON) -v $(RANDOM_TMP_DIR):/tmp -v $(HOME)/.ssh:/root/.ssh -v $(HOME)/.config:/root/.config $(CONTAINER_WHAT)
```

Set option to label volume mounts for SELinux:

```suggestion
	$(CONTAINER_COMMON) -v $(RANDOM_TMP_DIR):/tmp:Z -v $(HOME)/.ssh:/root/.ssh:Z -v $(HOME)/.config:/root/.config:Z $(CONTAINER_WHAT)
```

---

@csomh would be nice, but in some cases using `:Z` were not enough (Not sure if some home dirs has sometimes troubles with this option). Now I do not remember it exactly, but there were some issues. on the other hand, I've just moved duplicated part to makefile vars, so I did not changed it in sense of change of previous code-base, just deduplicated.